### PR TITLE
Ats2 how to's donotpublishyet

### PR DESCRIPTION
**Pull request by zwartd**

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
Hi @zwartd and @myrthewalhout, please sign the [CLA ](https://www.clahub.com/agreements/mendix/docs) if you have not already, this is required for merging pull requests. Thanks.

**Comment by zwartd** [comment]
Hi @Adam-Dupaski, 

This PR may also be published. I updated the screenshots and made some small changes to the text regarding a change in the application.

File: content/apidocs-mxsdk/mxsdk/creating-the-domain-model.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
 ---
```
Comment (Adam-Dupaski) [inline content/apidocs-mxsdk/mxsdk/creating-the-domain-model.md]: All of your linguistic/formatting changes here are to a version I already reviewed. Your linguistic/formatting changes are incorrect and need to be reverted to the state of my reviewed doc. Please make the changes and commit.

File: content/apidocs-mxsdk/mxsdk/creating-your-first-script.md [hunk 2]
```diff
@@ -1,13 +1,10 @@
 ---
```
Comment (Adam-Dupaski) [inline content/apidocs-mxsdk/mxsdk/creating-your-first-script.md]: All of your linguistic/formatting changes here are to a version I already reviewed. Your linguistic/formatting changes are incorrect and need to be reverted to the state of my reviewed doc. Changes to the parent doc also need to be reverted ("tutorial-for-the-mendix-sdk" is the correct parent). Please make the changes and commit.

File: content/apidocs-mxsdk/mxsdk/finding-things-in-the-model.md [hunk 3]
```diff
@@ -2,41 +2,38 @@
 title: "Finding Things in the Model"
```
Comment (Adam-Dupaski) [inline content/apidocs-mxsdk/mxsdk/finding-things-in-the-model.md]: All of your linguistic/formatting changes here are to a version I already reviewed. Your linguistic/formatting changes are incorrect and need to be reverted to the state of my reviewed doc. Please make the changes and commit.

File: content/apidocs-mxsdk/mxsdk/generating-code-from-the-model.md [hunk 4]
```diff
@@ -1,62 +1,61 @@
 ---
```
Comment (Adam-Dupaski) [inline content/apidocs-mxsdk/mxsdk/generating-code-from-the-model.md]: All of your linguistic/formatting changes here are to a version I already reviewed. Your linguistic/formatting changes are incorrect and need to be reverted to the state of my reviewed doc. Changes to the parent doc also need to be reverted ("tutorial-for-the-mendix-sdk" is the correct parent). Please make the changes and commit.

File: content/apidocs-mxsdk/mxsdk/index.md [hunk 5]
```diff
@@ -1,61 +1,59 @@
 ---
```
Comment (Adam-Dupaski) [inline content/apidocs-mxsdk/mxsdk/index.md]: All of your linguistic/formatting changes here are to a version I already reviewed. Your linguistic/formatting changes are incorrect and need to be reverted to the state of my reviewed doc. Please make the changes and commit.

File: content/apidocs-mxsdk/mxsdk/manipulating-existing-models.md [hunk 6]
```diff
@@ -3,9 +3,9 @@ title: "Manipulating Existing Models"
 parent: "tutorial-for-the-mendix-sdk"
```
Comment (Adam-Dupaski) [inline content/apidocs-mxsdk/mxsdk/manipulating-existing-models.md]: We capitalize titles of docs. Please recapitalize the titles in the links (like in the version that has already been reviewed) and commit the change.

File: content/apidocs-mxsdk/mxsdk/setting-up-your-development-environment.md [hunk 7]
```diff
@@ -1,16 +1,15 @@
 ---
```
Comment (Adam-Dupaski) [inline content/apidocs-mxsdk/mxsdk/setting-up-your-development-environment.md]: All of your linguistic/formatting changes here are to a version I already reviewed. Your linguistic/formatting changes are incorrect and need to be reverted to the state of my reviewed doc. Changes to the parent doc also need to be reverted ("tutorial-for-the-mendix-sdk" is the correct parent). Please make the changes and commit.

File: content/apidocs-mxsdk/mxsdk/your-learning-path-for-the-mendix-sdk.md [hunk 8]
```diff
@@ -0,0 +1,49 @@
+---
```
Comment (Adam-Dupaski) [inline content/apidocs-mxsdk/mxsdk/your-learning-path-for-the-mendix-sdk.md]: These changes are to a doc that has already been reviewed. Your linguistic/formatting changes are incorrect and need to be reverted. Please revert and commit.

File: content/ats/general/compatibility.md [hunk 9]
```diff
@@ -17,7 +17,7 @@ category: "General"
 ### Firefox
```
Comment (Adam-Dupaski) [inline content/ats/general/compatibility.md]: All of your linguistic/formatting changes here are to a version I already reviewed. Your linguistic/formatting changes are incorrect and need to be reverted to the state of my reviewed doc. Please make the changes and commit.

File: content/ats/howtos/ht-version-1/custom-action-general.md [hunk 10]
```diff
@@ -6,8 +6,6 @@ parent: "create-custom-actions"
 All the necessary information for the create-custom-action how-to's is found here:
```
Comment (Adam-Dupaski) [inline content/ats/howtos/ht-version-1/custom-action-general.md]: These changes are unnecessary - we write "How-to's." Please revert.

File: content/ats/howtos/ht-version-1/custom-action-howto-structure.md [hunk 11]
```diff
@@ -1,5 +1,5 @@
 ---
```
Comment (Adam-Dupaski) [inline content/ats/howtos/ht-version-1/custom-action-howto-structure.md]: Your language changes here are done to a version that has already been reviewed and are incorrect. Please revert.

File: content/ats/howtos/ht-version-1/custom-action-prerequisites.md [hunk 12]
```diff
@@ -1,5 +1,5 @@
 ---
```
Comment (Adam-Dupaski) [inline content/ats/howtos/ht-version-1/custom-action-prerequisites.md]: Your language changes here are done to a version that has already been reviewed. Your changes are incorrect. Please revert.

File: content/ats/howtos/ht-version-1/upload-file-using-ats-1.md [hunk 13]
```diff
@@ -1,5 +1,5 @@
 ---
```
Comment (Adam-Dupaski) [inline content/ats/howtos/ht-version-1/upload-file-using-ats-1.md]: Your change to the title is incorrect. Please revert.

File: content/ats/refguide/rg-version-1/checkbox-set-selector.md [hunk 14]
```diff
@@ -5,12 +5,12 @@ parent: "mendix-appstore-widgets-actions"
```
Comment (Adam-Dupaski) [inline content/ats/refguide/rg-version-1/checkbox-set-selector.md]: We cannot publish a doc with empty links like this. Please delete this doc from the PR or figure out a way to add the links.

File: content/ats/refguide/rg-version-1/standard-action-overview.md [hunk 15]
```diff
@@ -89,7 +89,11 @@ A _ConfirmationDialog_ is shown when a delete or microflow action requires a con
 | Action | Supported Widgets | Description |
```
Comment (Adam-Dupaski) [inline content/ats/refguide/rg-version-1/standard-action-overview.md]: I don't see the need for this change. Please revert.

File: content/community/documentation/content-writing-and-formatting-guidelines.md [hunk 16]
```diff
@@ -9,7 +9,7 @@ This document presents guidelines for creating a common voice, style, and struct
```
Comment (Adam-Dupaski) [inline content/community/documentation/content-writing-and-formatting-guidelines.md]: All of your language changes are incorrect. Please revert.

File: content/community/documentation/contribute-to-the-mendix-documentation.md [hunk 17]
```diff
@@ -18,7 +18,7 @@ All documentation contributions may undergo changes after editing according to o
 {{% /alert %}}
```
Comment (Adam-Dupaski) [inline content/community/documentation/contribute-to-the-mendix-documentation.md]: This is not a doc you can edit. Please remove from the PR.

File: content/community/documentation/the-how-to-template-page.md [hunk 18]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (Adam-Dupaski) [inline content/community/documentation/the-how-to-template-page.md]: This is not a doc you can edit. Please remove from the PR.

File: content/community/documentation/the-how-to-template-page.md [hunk 19]
```diff
@@ -1,6 +1,6 @@
...
-title: "How-to Template"
+title: "How-To Template"
 category: "Documentation"
```
Comment (Adam-Dupaski) [inline content/community/documentation/the-how-to-template-page.md]: This is not a doc you can edit. Please remove from the PR.

File: content/community/tools/the-mendix-forum.md [hunk 20]
```diff
@@ -40,7 +40,7 @@ Each question specifies the following elements:
 * The Mendix community members(s) who answered the question (**9**)
```
Comment (Adam-Dupaski) [inline content/community/tools/the-mendix-forum.md]: This is not a doc you can edit (and your change is linguistically incorrect). Please remove from the PR.

File: content/deployment/aws/index.md [hunk 21]
```diff
@@ -2,6 +2,5 @@
 title: "Amazon Web Services"
```
Comment (Adam-Dupaski) [inline content/deployment/aws/index.md]: Unnecessary change. Please revert.

File: content/deployment/azure/index.md [hunk 22]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (Adam-Dupaski) [inline content/deployment/azure/index.md]: Unnecessary change. Please revert.

File: content/deployment/azure/index.md [hunk 23]
```diff
@@ -1,6 +1,6 @@
...
 title: "Azure"
 ---
-## Documents in This Category
```
Comment (Adam-Dupaski) [inline content/deployment/azure/index.md]: Unnecessary change. Please revert.

File: content/deployment/mendixcloud/custom-error-page.md [hunk 24]
```diff
@@ -30,18 +30,18 @@ Before starting this how-to, make sure you have completed the following prerequi
 * Have a local repository of the app
```
Comment (Adam-Dupaski) [inline content/deployment/mendixcloud/custom-error-page.md]: Unnecessary changes. Please revert.

File: content/deployment/mendixcloud/how-to-deploy-a-mendix-app-on-amazon-web-services.md [hunk 25]
```diff
@@ -0,0 +1,188 @@
+---
```
Comment (Adam-Dupaski) [inline content/deployment/mendixcloud/how-to-deploy-a-mendix-app-on-amazon-web-services.md]: I don't know why this doc is in this PR. Please remove.

File: content/deployment/mendixcloud/how-to-deploy-a-mendix-app-on-azure.md [hunk 26]
```diff
@@ -0,0 +1,224 @@
+---
```
Comment (Adam-Dupaski) [inline content/deployment/mendixcloud/how-to-deploy-a-mendix-app-on-azure.md]: I don't know why this doc is in this PR. Please remove.

File: content/deployment/mendixcloud/index.md [hunk 27]
```diff
@@ -1,14 +1,11 @@
 ---
```
Comment (Adam-Dupaski) [inline content/deployment/mendixcloud/index.md]: Unnecessary changes, please revert.

File: content/deployment/mendixcloud/mxbuild.md [hunk 28]
```diff
@@ -1,6 +1,6 @@
 ---
```
Comment (Adam-Dupaski) [inline content/deployment/mendixcloud/mxbuild.md]: Unecessary changes, please revert.

File: content/deployment/mendixcloud/mxcloudv4.md [hunk 29]
```diff
@@ -75,6 +75,7 @@ There are some features missing in v4. Mendix will implement the following featu
```
Comment (Adam-Dupaski) [inline content/deployment/mendixcloud/mxcloudv4.md]: If you want to make a change to a non-Mansystems doc, please send a separate PR, because the change will have to be verified by R&D. Please remove this doc from this PR.

File: content/deployment/mendixcloud/request-handlers-to-pbar.md [hunk 30]
```diff
@@ -0,0 +1,38 @@
+---
```
Comment (Adam-Dupaski) [inline content/deployment/mendixcloud/request-handlers-to-pbar.md]: I don't see why this doc is in this PR. Please remove.

Comment (Adam-Dupaski) [inline content/deployment/mendixcloud/request-handlers-to-pbar.md]: I don't see why this doc is in this PR. If you want to make a change to a non-Mansystems doc, please send a separate PR, because the change will have to be verified by R&D. Please remove this doc from this PR.

File: content/deployment/mendixcloud/security-constraints-in-the-mendix-cloud.md [hunk 31]
```diff
@@ -0,0 +1,19 @@
+---
```
Comment (Adam-Dupaski) [inline content/deployment/mendixcloud/security-constraints-in-the-mendix-cloud.md]: I don't see why this doc is in this PR. If you want to make a change to a non-Mansystems doc, please send a separate PR, because the change will have to be verified by R&D. Please remove this doc from this PR.

File: content/deployment/on-premises/design-the-architecture.md [hunk 32]
```diff
@@ -13,39 +13,43 @@ The options below are just a few examples of the most commonly used architecture
```
Comment (Adam-Dupaski) [inline content/deployment/on-premises/design-the-architecture.md]: I don't see why this doc is in this PR. If you want to make a change to a non-Mansystems doc, please send a separate PR, because the change will have to be verified by R&D. Please remove this doc from this PR.

File: content/deployment/on-premises/restoring-a-sql-server-database.md [hunk 33]
```diff
@@ -1,5 +1,5 @@
 ---
```
Comment (Adam-Dupaski) [inline content/deployment/on-premises/restoring-a-sql-server-database.md]: Unnecessary and incorrect change. Please remove this doc from this PR.

File: content/deployment/on-premises/troubleshooting.md [hunk 34]
```diff
@@ -3,8 +3,7 @@ title: "Troubleshooting"
 category: "On-Premises"
```
Comment (Adam-Dupaski) [inline content/deployment/on-premises/troubleshooting.md]: If you want to make a change to a non-Mansystems doc, please send a separate PR, because the change will have to be verified by R&D. Please remove this doc from this PR.

File: content/deployment/sap-cloud-platform/deploy-a-mendix-app-to-sap-cloud-platform.md [hunk 35]
```diff
@@ -7,24 +7,23 @@ tags: ["SAP", "SAP Cloud Platform", "on-premises", "Cloud Connector"]
```
Comment (Adam-Dupaski) [inline content/deployment/sap-cloud-platform/deploy-a-mendix-app-to-sap-cloud-platform.md]: If you want to make a change to a non-Mansystems doc, please send a separate PR, because the change will have to be verified by R&D. Please remove this doc from this PR.

File: content/developerportal/collaborate/feedback.md [hunk 36]
```diff
@@ -37,8 +37,8 @@ You can click the following buttons:
```
Comment (Adam-Dupaski) [inline content/developerportal/collaborate/feedback.md]: If you want to make a change to a non-Mansystems doc, please send a separate PR, because the change will have to be verified by R&D. Please remove this doc from this PR.

File: content/developerportal/develop/model.md [hunk 37]
```diff
@@ -28,9 +28,9 @@ Follow these steps to quickly start developing your new app:
```
Comment (Adam-Dupaski) [inline content/developerportal/develop/model.md]: If you want to make a change to a non-Mansystems doc, please send a separate PR, because the change will have to be verified by R&D. Please remove this doc from this PR.

File: content/developerportal/general/twofactor-authenticator.md [hunk 38]
```diff
@@ -11,7 +11,7 @@ In the **Deploy** and **Operate** categories of the [Developer Portal](http://ho
```
Comment (Adam-Dupaski) [inline content/developerportal/general/twofactor-authenticator.md]: Please revert this typo.

File: content/developerportal/howto/deactivate-users.md [hunk 39]
```diff
@@ -49,8 +49,7 @@ To manage the users in your company, follow these steps:
```
Comment (Adam-Dupaski) [inline content/developerportal/howto/deactivate-users.md]: Unnecessary change, please revert.

File: content/developerportal/howto/delete-apps.md [hunk 40]
```diff
@@ -1,38 +1,83 @@
 ---
```
Comment (Adam-Dupaski) [inline content/developerportal/howto/delete-apps.md]: If you want to make a change to a non-Mansystems doc, please send a separate PR, because the change will have to be verified by R&D. Please remove this doc from this PR.

**Comment by Adam-Dupaski** [comment]
This PR has many problems. I left comments identifying a lot of them. Firstly, 687 files in one PR is way too many to merge at once; there will be innumerable conflicts, most of which can only be fixed on your end. Please make PRs much smaller in the future. I would like to see this PR broken up into separate smaller PRs for better management. Secondly, all of the non-Mansystems and non-ATS docs should be removed from this PR altogether. If you want to submit changes on such docs, they need to be in individual PRs, because that would require verification from R&D. Most of the changes you are making here are incorrect from a style guide/language/formatting perspective, and potentially incorrect on a technical level for any technical content. Please make these changes, otherwise this PR cannot be merged. @zwartd @myrthewalhout

**Comment by zwartd** [comment]
Hi @Adam-Dupaski,

Sorry I made a mistake regarding the update process. I created a new PR 976 to replace this one. It contains the exact same ATS 2 files as this PR only the PR is up to date with the Mendix development branche.

**Comment by Adam-Dupaski** [comment]
@zwartd #976 was merged. This PR can be closed, correct?
